dNavigation/dTerrain/RawMesh.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "NiPoint3.h"
```

Comment:
I believe you can forward declare NiPoint3 here as well, but not sure since its in a vector.